membership.workflows.yml

```diff
@@ -16,3 +16,35 @@ membership_default:
       label: Restart
       from: [expired]
       to: active
+
+membership_term_default:
+  id: membership_term_default
+  label: Default Membership Term Workflow
```

Comment:
This isn't so much a workflow for a membership term as much as a term-limited membership itself. Maybe "Default Term-Limited Membership Workflow" would be clearer?